omero/developers/Python.txt

```diff
@@ -781,12 +781,6 @@ ROIs
     conn.connect()
     updateService = conn.getUpdateService()
 
--  **Configuration**
-
-::
-
-    imageId = 27544
```

Comment:
Scracth this comment, I looked at the chnage in isolation rather then checking the whole file.